lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -152,8 +152,6 @@ class SemanticsTextEditingStrategy extends DefaultTextEditingStrategy {
       {OnChangeCallback? onChange, OnActionCallback? onAction}) {
     isEnabled = true;
     inputConfiguration = inputConfig;
-    onChange = onChange;
-    onAction = onAction;
```

Comment:
Was this just to mark these as "used" parameters?

Reply:
I was wondering that as well...